core/src/main/java/io/questdb/cairo/TxReader.java

```diff
@@ -419,6 +419,40 @@ protected long unsafeReadFixedRowCount() {
     }
 
     protected int unsafeReadSymbolWriterIndexOffset(int denseSymbolIndex) {
-        return getInt(getSymbolWriterIndexOffset(denseSymbolIndex));
+        return getInt(TableUtils.getSymbolWriterIndexOffset(denseSymbolIndex));
+    }
+
+    public void dumpTo(MemoryCMARW mem) {
```

Comment:
```suggestion
    public void dumpTo(MemoryW mem) {
```

Reply:
Addressed in 3e2f0b4cea1601b70e737b15316c1f8e85b8a3b9